testgrid/config.yaml

```diff
@@ -4275,14 +4275,14 @@ dashboards:
   - name: crio-e2e-fedora
     test_group_name: test_pull_request_crio_e2e_fedora
     open_test_template:
-        url: https://gcsweb-ci.svc.ci.openshift.org/gcs/<gcs_prefix>/<changelist>.txt
+        url: https://gcsweb-ci.svc.ci.openshift.org/gcs/<gcs_prefix>/<changelist>/artifacts/artifacts/junit_01.xml
```

Comment:
looks like this needs to be changed back indeed

Reply:
correct gs link looks like this: https://storage.googleapis.com/origin-federated-results/pr-logs/directory/test_pull_request_crio_e2e_rhel/1205.txt

Reply:
actually, we're still missing something https://openshift-gce-devel.appspot.com/jobs/origin-ci-test/pr-logs/pull/kubernetes-incubator_cri-o/ (/cc @stevekuznetsov )